com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/listeners/ErrorProcessListener.java

```diff
@@ -27,8 +27,8 @@ public void onEvent(FlowableEvent event) {
         handler.handle(event);
 
         if (event instanceof FlowableEngineEvent) {
-            ClientReleaser clientReleaser = new ClientReleaser((FlowableEngineEvent) event, clientProvider);
-            clientReleaser.releaseClient();
+            ClientReleaser clientReleaser = new ClientReleaser(clientProvider);

```

Comment:
Extract the body of this if in a separate method. That way, you'll reduce the explosion of braces (), which now exists because of the casting, and you'll reduce the number of things this method does (which is always a good thing :)).